csvcubed/csvcubed/schema/cube-config-schema.jsonc

```diff
@@ -0,0 +1,430 @@
+{
+	"$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "http://gss-cogs.github.io/gss-cogs/csvcubed/csvcubed/schemas/cube-config-schema.jsonc",
+    "title": "Schema for a qb-config cube",
+    "description": "Schema to be used for creation & validation of a single csv-cubed-flavoured linked data cube in CSV-W format",
+    "type": "object",
+
+    "properties": {
+
+		"publisher": {
+            "description": "The schema publisher link uri",
+            "type": "string",
+            "format": "uri"
+		},
+
+        "creator": {
+            "description": "The schema publisher link uri",
+            "type": "string",
+            "format": "uri"
+        },
+
+        "spatial_bound": {
+            "description": "URI that defines the spatial / geographic bounds of the data contained herein",
+            "type": "string",
+            "format": "uri",
+            "const": "https://tdg.docbook.org/tdg/publishers/5.1/dcterms.html#dcterms.spatial"
+        },
+
+        "temporal_bound": {
+            "description": "URI that defines the temporal bounds of the data contained herein",
+            "type": "string",
+            "format": "uri",
+            "const": "https://tdg.docbook.org/tdg/publishers/5.1/dcterms.html#dcterms.temporal"
+        },
+
+        "columns": {
```

Comment:
`columns` should be a dictionary of column name to the object type.

Reply:
Yes, I've talked with @Giraffe-Technology and we've got a better handle on this one now.

Reply:
Resolved, the column name can be anything (string), on adding the 'type' property an enum of the available 'columns' is available.  This offers dimension, attribute, measure, unit and observation.
Note: these values are dynamically generated from the type: const: string values.